api/v1beta1/awscluster_conversion.go

```diff
@@ -134,6 +150,10 @@ func restoreControlPlaneLoadBalancer(restored, dst *infrav2.AWSLoadBalancerSpec)
 	dst.DisableHostsRewrite = restored.DisableHostsRewrite
 	dst.PreserveClientIP = restored.PreserveClientIP
 	dst.IngressRules = restored.IngressRules
+	dst.AdditionalSecurityGroups = restored.AdditionalSecurityGroups
+	dst.Scheme = restored.Scheme
+	dst.CrossZoneLoadBalancing = restored.CrossZoneLoadBalancing
+	dst.Subnets = restored.Subnets
```

Comment:
Adding these fields caused conversion tests to pass, but it seems incorrect if they were missing for the control plane before.